### PR TITLE
Update templates to make mitigations faster and enable external users to report more easily

### PR DESCRIPTION
**Pull request by laghee**
**Asana Task/Github Issue:**
https://app.asana.com/0/1201870266890790/1207370685063605/f

## Description
As discussed, we're removing the "create an issue" step from our regular mitigation flow to make it a bit more efficient. To that end I updated the PR template to explicitly prompt for the information we want the PR description to contain and adapted the issue template to be more user-friendly for potential non-internal folks reporting (should that ever occur 😉).

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ If you'd like to license the source for commercial use, [please reach out](https
 
 ## Questions
 
+- **Where can I find out why a particular exception has been added?** We mostly do two kinds of mitigations -- either allowing [certain trackers](features/tracker-allowlist.json) or disabling one or more [protections](features/) on specific URLs or on certain [platforms](overrides). If you search in Code for the domain you're interested in, in most cases there will be a `"reason"` key below the exception for it with the URL of either an issue (old process) or a pull request (current process). Within the description of that linked URL we explain why we've added a breakage mitigation to the site in question and exactly what we're allowing or disabling to improve users' experience on the page.
```
Comment (SlayterDev) [inline README.md:58]: @laghee have you been able to verify these links work (specifically protections and platforms)? Just want to be sure before we merge.

Comment (laghee) [reply README.md:58]: Ehrm, no actually 😬 They look... weirdly specific to my branch, but I was hoping that was just an in-flux formatting artifact. I could replace with full URLs -- maybe that's more foolproof?

Comment (SlayterDev) [reply README.md:58]: I suppose let's use full urls to be safe 👍

Comment (laghee) [reply README.md]: Updated!

**Review by SlayterDev — APPROVED** [review]
LGTM!

**Pull request merged**